### PR TITLE
update dataset version manually + tried to do it in js

### PR DESCRIPTION
**Pull request by vviers**
@martinratinaud I'm not very good with JS and so I updated the dataset manually in the interest of time

Please feel free to take a look at what I did in the JS file and see if I was on the right track ?

### CODE REVIEW

File: assets/js/open-terms-archive/scripta-manent.js [hunk 1]
```diff
@@ -62,7 +67,12 @@ document.addEventListener("DOMContentLoaded", () => {
 	}
 
 	async function getServices() {
-		const request = new Request(APIBaseURL+'/list_services/v1/?multiple_versions_only=true', requestHeaders);
+		const request = new Request(APIBaseURL + '/list_services/v1/?multiple_versions_only=true', requestHeaders);
+		return async_fetch(request);
+	}
+
+	async function getDatasetVersion() {
+		const request = new Request(APIBaseURL + '/version', requestHeaders);
```
Comment (martinratinaud) [inline assets/js/open-terms-archive/scripta-manent.js:75]: isn't it a key inside this call ?

File: assets/js/open-terms-archive/scripta-manent.js [hunk 2]
```diff
@@ -281,11 +291,11 @@ document.addEventListener("DOMContentLoaded", () => {
 			firstDocumentDate: firstDocumentDate,
 			secondDocumentDate: secondDocumentDate,
 			firstDocumentVersionAtDate: firstDocumentVersionAtDate,
-			secondDocumentVersionAtDate: secondDocumentVersionAtDate,Date,
+			secondDocumentVersionAtDate: secondDocumentVersionAtDate, Date,
```
Comment (martinratinaud) [inline assets/js/open-terms-archive/scripta-manent.js:294]: this is a weird indetation :-)

**Review by martinratinaud — COMMENTED** [review]
This looks like a good start

File: assets/js/open-terms-archive/scripta-manent.js (hunk 1 above)
Comment (vviers) [reply assets/js/open-terms-archive/scripta-manent.js:75]: the API "version" is 3 things - dataset_url, dataset_date, and the commit_sha it was build with

File: assets/js/open-terms-archive/scripta-manent.js (hunk 2 above)
Comment (vviers) [reply assets/js/open-terms-archive/scripta-manent.js:294]: mmh that's probably because of my formatter in VSCode

**Review by martinratinaud — APPROVED** [review]

**Pull request merged**